### DISCUSSION TITLE
Just finished!

### DISCUSSION THREAD

**Discussion by doc1011** [CircuitSetup/Time-Circuits-Display#33] [Show and tell]
Loved working on this kit. Printed in PETG. Works great :) 

![IMG_6854](https://user-images.githubusercontent.com/42011383/215620878-2a4f726b-7269-4c71-95f8-94d21d0e9b9e.jpeg)

**Comment by CircuitSetup** [comment]
Very nice paint job!  How was sanding in PETG? I haven't printed the TCD with it yet.

**Reply by doc1011** [reply]
Cheers mate - PETG worked great - took some time and a bit of filler here and there but the results were fantastic. I used some filler primer as well which helped. I've put myself on the waiting list for another kit ;)

**Comment by Ricky3Nut** [comment]
Very nice work! I built mine probably over a year and a half ago and I just now finally got a laptop stand to display mine. But the platform is just about a half inch too narrow so I might try and find something a little bigger. And I need to figure out how to install all of the updates that have been released.

It would be nice if the power feed came into the top side or even the back of the keypad. Mine comes into the front and it's in an awkward place to hide. I want to print a little sloped stand for the keypad, probably in TPU.  

I used PLA for my kit and a hammered paint, but I might have to do a little rework on it to clean up some cracks. My keypad doesn't look anywhere as nice as yours! I'll try and share some photos soon.

Congrats!! 



Get BlueMail for Android
